src/components/tables/Captures/Empty.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Typography } from '@mui/material';
+import ExternalLink from 'components/shared/ExternalLink';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+function Empty() {
```

Comment:
Not sure how I feel about this pattern... but might be useful in the future when we have a more fleshed out table. We could just pass an "Empty Component" to the table.